platform/default/mbgl/storage/offline_download.cpp

```diff
@@ -107,6 +107,7 @@ OfflineRegionStatus OfflineDownload::getStatus() const {
 
         case SourceType::Video:
         case SourceType::Annotations:
+        case SourceType::Image:
             break;
```

Comment:
We'll need to add offline support for image sources -- it should download and store the image in the offline database.